test/integration/publication_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class PublicationTest < ActionDispatch::IntegrationTest
   test "random but valid items do not error" do
-    setup_and_visit_random_content_item(document_type: 'statutory_guidance')
+    setup_and_visit_random_content_item(frontend_schema: 'statutory_guidance')
```

Comment:
@fofr Following our discussion, I've renamed the `document_type` param to `frontend_schema` since that's what it's actually used for. This makes it clearer what we're requesting as a random example.